docs/sagemaker/source/dlcs/features.md

```diff
@@ -0,0 +1,29 @@
+# Features & benefits
+
+The Hugging Face DLCs provide ready-to-use, tested environments to train and deploy Hugging Face models.
+
+## One command is all you need
+
+With the new Hugging Face DLCs, train and deploy cutting-edge Transformers-based NLP models in a single line of code. The Hugging Face PyTorch DLCs for training come with all the libraries installed to run a single command e.g. via TRL CLI to fine-tune LLMs on any setting, either single-GPU, single-node multi-GPU, and more.
+
+## Accelerate machine learning from science to production
+
+In addition to Hugging Face DLCs, we created a first-class Hugging Face library for inference, huggingface-inference-toolkit, that comes with the Hugging Face PyTorch DLCs for inference, with full support on serving any PyTorch model on AWS.
```

Comment:
`sagemaker-huggingface-inference-toolkit` + link to it https://github.com/aws/sagemaker-huggingface-inference-toolkit/tree/main/src/sagemaker_huggingface_inference_toolkit